tests/php/Authenticator/ChangePasswordHandlerTest.php

```diff
@@ -36,6 +36,16 @@ protected function setUp(): void
             ->set(Member::class, 'auto_login_token_lifetime', 10);
     }
 
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        foreach (['tempHashAlreadyGenerated', 'tempHashAlreadyProcessed'] as $property) {
+            $refl = new ReflectionProperty(SecurityChangePasswordHandler::class, $property);
+            $refl->setAccessible(true);
+            $refl->setValue(null, false);
+        }
```

Comment:
```suggestion
        foreach (['tempHashAlreadyGenerated', 'tempHashAlreadyProcessed'] as $property) {
            $refl = new ReflectionProperty(SecurityChangePasswordHandler::class, $property);
            $refl->setAccessible(true);
            $refl->setValue(null, false);
        }
        parent::tearDown();
```

As per https://github.com/silverstripe/silverstripe-framework/blob/01461ff740b42e7c372f685bda1cd69a2ce27e6f/src/Dev/SapphireTest.php#L594
> User code should call parent::tearDown() after custom tear down code

Reply:
Done